diff --git a/dist/node/chunks/dep-M1IYMR16.js b/dist/node/chunks/dep-M1IYMR16.js
index 6de6e61108137b1c9491e37dad7834815d247ed2..b4c6bd9a8dfab8e0150b1bcb19b16c14fbfbf75b 100644
--- a/dist/node/chunks/dep-M1IYMR16.js
+++ b/dist/node/chunks/dep-M1IYMR16.js
@@ -53775,7 +53775,7 @@ async function resolveConfig(inlineConfig, command, defaultMode = "development",
     // at least 64bits is recommended
     // https://owasp.org/www-community/vulnerabilities/Insufficient_Session-ID_Length
     webSocketToken: Buffer.from(
-      crypto$2.getRandomValues(new Uint8Array(9))
+      crypto$2.webcrypto.getRandomValues(new Uint8Array(9))
     ).toString("base64url"),
     getSortedPlugins: undefined,
     getSortedPluginHooks: undefined,
